regparser/tree/xml_parser/reg_text.py

```diff
@@ -92,12 +106,16 @@ def build_tree(reg_xml):
 
     return tree
 
-
 def get_subpart_title(subpart_xml):
     hds = subpart_xml.xpath('./RESERVED|./HD')
     if hds:
         return [hd.text for hd in hds][0]
 
+def get_subjgrp_title(subjgrp_xml):
+    hds = subjgrp_xml.xpath('./RESERVED|./HD')
+    if hds:
+        return [hd.text for hd in hds][0]
```

Comment:
Looks like this is a copy-aste of `get_subpart_title`. Would it be better to rename that fn and reuse?